### PR TITLE
Pecos aniso viscosity

### PR DESCRIPTION
**Pull request by clarkpede**
This is the hybrid portion of the SU2.  I haven't yet checked to make sure that my merge from `pecos-dev` into `pecos-aniso_viscosity` hasn't changed the CDP vs SU2 comparison.  I can do that sometime on Thursday.

The [documentation on the SVN repo](https://svn.ices.utexas.edu/repos/turbulence/docs/SU2_hybrid/) contains some information on the changes.  It's a little out-of-date, but there haven't been any revolutionary changes on the implementation side of things.  I mostly need to update what we've continued to implement (v2-f now supported, more outputs written to files, etc.).

Side note: Valgrind shows some false positives in the MKL side of the eigenvector calculations.  See [this thread on their forums](https://software.intel.com/en-us/forums/intel-math-kernel-library/topic/279131) for more information.  It seems to be a nonissue.

### CODE REVIEW

**Comment by clarkpede** [comment]
So I've run into a snag.

Previously on SU2, the linear solver could run to the max iterations and then spit out an erroneous output with no warnings or errors.  While developing the hybrid method, I put in a check, error message, and force exit.  In time-stepping, I don't think we want the linear solver to fail to converge.

The only problem is that a lot of the regression tests are set up with too few iterations on the linear solvers.  In the last CI test, a lot of them errored out because the linear solver didn't converge.

So I think we have three options:

1. I change the error to a warning, and not have it exit the program.
2. I leave it as an error and update the regression tests to have reasonable linear solver iteration limits.  While this seems to be the best solution, it will be a dreadful mess.  If more iterations are used, the residuals might be different.  So changing the number of iterations means we would have to update the test values for _every single test_ that is currently maxing out the iterations.
3. I include an extra config option to toggle the Warning/Error behavior.  It would be set to "False" by default, and would only display warnings if the linear solver exceeds its iterations.  But we could also tell the code to error out if the linear solver doesn't converge by setting this option to "True".

I like the 3rd option best, but I'm implementing the option 1 first as a quick fix. What do you all think?

**Comment by clarkpede** [comment]
I've attached some updated plots comparing the SU2's current state with CDP.  The data for CDP are from our simple test problem from a month ago.
r_k
r_k_large
w_rans_large

**Comment by trevilo** [comment]
> I like the 3rd option best, but I'm implementing the option 1 first as a quick fix. What do you all think?

I agree that option 3 is best, but option 1 is fine for now.

**Comment by clarkpede** [comment]
Alright.  I think I squashed all of the bugs. There may be one or two issues still, but we're matching CDP and the regression tests all check out. Review or merge whenever you want.

File: SU2_CFD/src/driver_structure.cpp [hunk 1]
```diff
@@ -1449,9 +1529,100 @@ void CDriver::Numerics_Preprocessing(CNumerics ****numerics_container,
         numerics_container[iMGlevel][TURB_SOL][CONV_BOUND_TERM] = new CUpwSca_TurbKE(nDim, nVar_Turb, config);
         numerics_container[iMGlevel][TURB_SOL][VISC_BOUND_TERM] = new CAvgGrad_TurbKE(nDim, nVar_Turb, constants, false, config);
       }
+      else if (zetaf_ke) {
```
Comment (trevilo) [inline SU2_CFD/src/driver_structure.cpp]: Looks like the `else if (zetaf_ke)` block got duplicated here.

**Review by trevilo — APPROVED** [review]
Looks good @clarkpede.  I haven't had time to review everything in detail, but I vote you go ahead with the merge whenever you're ready.  I'll look things over more closely as I start using the code.

**Comment by clarkpede** [comment]
@trevilo @shaering Just so you know, I've updated the [SVN documentation](https://svn.ices.utexas.edu/repos/turbulence/docs/SU2_hybrid/) to reflect the more recent uses in the code.  But if you are having trouble understanding or using the code, please let me know.

**Pull request merged**